watertap/core/zero_order_costing.py

```diff
@@ -0,0 +1,333 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+General costing package for zero-order processes.
+"""
+import pyomo.environ as pyo
```

Comment:
I know this isn't a big deal, but in WaterTAP and most IDAES models (I think) we generally import the objects from pyomo environ rather than use pyo.Var() for example.

Reply:
This is the standard form Pyomo themselves use for their documentation. We might want to standardize on that form for public facing code to help users.